newIDE/app/src/stories/componentStories/DismissableTutorialMessage.stories.js

```diff
@@ -5,78 +5,95 @@ import { action } from '@storybook/addon-actions';
 import muiDecorator from '../ThemeDecorator';
 import paperDecorator from '../PaperDecorator';
 
-import { DismissableTutorialMessageWidget } from '../../Hints/DismissableTutorialMessage';
+import DismissableTutorialMessage from '../../Hints/DismissableTutorialMessage';
 import { type Tutorial } from '../../Utils/GDevelopServices/Tutorial';
-import { initialPreferences } from '../../MainFrame/Preferences/PreferencesContext';
+import {
+  initialPreferences,
+  type Preferences,
+} from '../../MainFrame/Preferences/PreferencesContext';
+import PreferencesContext from '../../MainFrame/Preferences/PreferencesContext';
+import { TutorialContext } from '../../Tutorial/TutorialContext';
+
+const defaultTutorials = [
+  {
+    id: 'tutorial-1',
+    title: 'Tutorial 1',
+    description: 'Description 1',
+    thumbnailUrl:
+      'https://raw.githubusercontent.com/4ian/GDevelop/master/Core/docs/images/gdlogo.png',
+    link: 'https://example.com/tutorial.html',
+    type: 'video',
+  },
+  {
+    id: 'tutorial-2',
+    title: 'Tutorial 2',
+    description: 'Description 2',
+    thumbnailUrl:
+      'https://raw.githubusercontent.com/4ian/GDevelop/master/Core/docs/images/gdlogo.png',
+    link: 'https://example.com/tutorial.html',
+    type: 'text',
+  },
+];
+
+type Props = {|
+  tutorials?: ?Array<Tutorial>,
+  preferences?: Preferences,
+  tutorialId: string,
+|};
+
+const WrappedDismissableTutorialMessage = ({
```

Comment:
Excellent, it makes sense to remove the complexity from the components and instead have this complexity in the stories (and if multiple stories are using the same fake data/providers, it's easy later to just refactor these).